server/server.go

```diff
@@ -295,26 +294,8 @@ func (s *Server) restore(ctx context.Context) []string {
 	return imagesOfDeletedContainers
 }
 
-// cleanupSandboxesOnShutdown Remove all running Sandboxes on system shutdown
-func (s *Server) cleanupSandboxesOnShutdown(ctx context.Context) {
-	_, err := os.Stat(shutdownFile)
-	if err == nil || !os.IsNotExist(err) {
-		log.Debugf(ctx, "Shutting down all sandboxes, on shutdown")
-		s.stopAllPodSandboxes(ctx)
```

Comment:
Isn't this function unused after that and can be removed as well?

Reply:
good point! fixed